### PR TITLE
Move language selector to footer

### PR DESCRIPTION
**Pull request by mllocs**
http://g.recordit.co/Qz7JruPxF9.gif

### CODE REVIEW

**Comment by mllocs** [comment]
[Move language selector to the footer](https://trello.com/c/Pc3Ceq6n/125-move-language-selector-to-the-footer)

File: app/views/application/menus/_language_switcher.html.erb [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
+<div class="dropup language-selector">
+  <a class="btn btn-default dropdown-toggle language" type="button" id="dropdownLanguage" data-toggle="dropdown">
+    <span><%= t("locales.#{locale}") %></span>
+    <b class="caret"></b>
+  </a>
+  <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenu1" >
+    <% I18n.available_locales.each do |locale| %>
+      <% if I18n.locale != locale %>
+        <li>
+          <%= link_to t("locales.#{locale}"), switch_lang_path(locale: locale) %>
```
Comment (markets) [inline app/views/application/menus/_language_switcher.html.erb]: Maybe we should use `t("locales.#{locale}", locale: locale)` here?

Comment (mllocs) [reply app/views/application/menus/_language_switcher.html.erb]: why? that wouldn't be the same result. BTW, I just cut and pasted from one place to a separated file :)

Comment (markets) [reply app/views/application/menus/_language_switcher.html.erb]: ah yes @mllocs! this is a proposal change, I think it's more natural for the users to see the language in their language.

Comment (sauloperez) [reply app/views/application/menus/_language_switcher.html.erb]: I agree too. Should it go in a different PR though?

Comment (markets) [reply app/views/application/menus/_language_switcher.html.erb]: Not sure guys, in this case, if a different PR is necessary, with such a small change... `¯\_(ツ)_/¯`. I'm totally +1 for small PRs, but now that we're touching the languages selector, maybe it's not a bad idea to add this change (no collateral effects) in this branch as well.

File: app/views/application/menus/_language_switcher.html.erb [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
+<div class="dropup language-selector">
+  <a class="btn btn-default dropdown-toggle language" type="button" id="dropdownLanguage" data-toggle="dropdown">
+    <span><%= t("locales.#{locale}") %></span>
+    <b class="caret"></b>
+  </a>
+  <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenu1" >
+    <% I18n.available_locales.each do |locale| %>
+      <% if I18n.locale != locale %>
+        <li>
+          <%= link_to t("locales.#{locale}", locale: locale), switch_lang_path(locale: locale) %>
```
Comment (mllocs) [inline app/views/application/menus/_language_switcher.html.erb]: @markets 😉 
I didn't understand you the first time, I changed it 💪

**Comment by enricostano** [comment]
Deployed to https://staging.timeoverflow.org/

cc/ @mllocs @sseerrggii

**Comment by markets** [comment]
@enricostano @mllocs I just changed the base branch pointing to `develop` (`feature/nicer-mobile-views` have been merged)

**Comment by enricostano** [comment]
> @enricostano @mllocs I just changed the base branch pointing to develop (feature/nicer-mobile-views have been merged)

Oh! I didn't noticed it wasn't pointing to `develop`, we should add that to the PR description. thanks @markets !

**Pull request merged**